sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonExternalTransform.java

```diff
@@ -266,6 +277,14 @@ public PythonExternalTransform<InputT, OutputT> withOutputCoder(Coder<?> outputC
     return this;
   }
 
+  public PythonExternalTransform<InputT, OutputT> withExtraPackages(List<String> extraPackages) {
```

Comment:
Please add a Java doc (for example, we should mention that this can also include the version).